src/java/io/bazel/rulesscala/jar/JarCreator.java

```diff
@@ -173,6 +173,16 @@ public void execute() throws IOException {
     }
   }
 
+  @Override
+  protected boolean ignoreFileName(String name) {
+    /*
+     * It does not make sense to copy signature files
```

Comment:
Do you think it would be wise to rename `JarCreator` to `FatJarCreator` / `DeployJarCreator` / `UberJarCreator`? Current name made me wonder why the override is like this and the base is just "false". Until I read the comment and remembered